optax/_src/alias.py

```diff
@@ -992,11 +992,27 @@ def noisy_sgd(
     gamma: float = 0.55,
     seed: int = 0
 ) -> base.GradientTransformation:
-  r"""A variant of SGD with added noise.
```

Comment:
First line needs to end with ".", so I would leave the first line as it is, and add your description below

Reply:
and the line after should be blank, so something like:

```
r"""A variant of SGD with added noise.

Noisy SGD is ...
"""

Reply:
Could you clarify if the requirement for the first line to end with "." is driven by stylistic guidelines or technical reasons? The list of optimizers at the beginning of the documention isn't effected and displays the first sentence of the docstring correctly.

Reply:
its a requirement of the python style: https://peps.python.org/pep-0257/#multi-line-docstrings

Internally we get an error when importing the code if the docstring doesn't follow this convention

Reply:
you're right though that this would be something worth adding to https://optax.readthedocs.io/en/latest/development.html